ccmlib/scylla_cluster.py

```diff
@@ -22,13 +23,19 @@ def __init__(self, path, name, partitioner=None, install_dir=None,
                  create_directory=True, version=None, verbose=False,
                  force_wait_for_cluster_start=False, manager=None, **kwargs):
         install_func = common.scylla_extract_install_dir_and_mode
-        install_dir, self.scylla_mode = install_func(install_dir)
+
+        cassandra_version = kwargs.get('cassandra_version', version)
+        if kwargs.get('cassandra_version', version):
```

Comment:
nit - if we extracted it already why not use cassandra_version ?

Reply:
hmmm, a disturbance in the force. I'll rectify that

Reply:
fixed this